lib/vestal_versions/users.rb

```diff
@@ -23,29 +26,20 @@ def version_attributes
 
   # Instance methods added to VestalVersions::Version to accomodate incoming user information.
   module UserVersionMethods
-    extend ActiveSupport::Concern
```

Comment:
Do we not need to extend this?

Reply:
Since we're prepending instead of including, I'm not sure ActiveSupport::Concern even does anything here. It shouldn't be needed in any case.